### DISCUSSION TITLE
[@storybook/web-components] Why isn't my component being imported?

### DISCUSSION THREAD

**Discussion by jmcgavin** [storybookjs/storybook#17078] [Help]
I have the following story:

```js
import { html } from 'lit-element'
import '../src/my-component.js'

export default {
  title: 'My Component',
}

const Template = ({ title }) => {
  return html`
    <my-component
      .title=${title}>
    </my-component>
  `
}

export const Default = (args) => Template(args)
Default.args = {
  title: 'Hello world'
}
```

The Storybook DOM shows an empty `<my-component>` element without a shadow-root. I've put a breakpoint in `../src/my-component.js` and I never reach it. It's as though the import of my custom component (which also does the `window.customElements.define()` is not working. Any ideas?

**Comment by ballcoach12** [comment]
I  get the same result on a similar project.

**Comment by break-stuff** [comment]
It sounds like your component isn't being registered in the DOM. How are you doing that?

**Comment by break-stuff** [comment]
Oh, I just realized this is 2 years old!!! I hope you found an answer. 😬
